test/compilable/imports/cstuff1.c

```diff
@@ -289,14 +289,38 @@ void test4(int i)
     long long ll = 12L;
     ll = 12UL;
     long double ld = 1.0L;
-    char* s = "hello" "betty";
+    const char* s = "hello" "betty";
 }
 
 /********************************/
 
-void test__func__()
```

Comment:
I don't think you meant to remove this test.

Reply:
good catch. Rebasing is a minefield.